src/core/parser/component.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  transformStylesAssignmentsToJson,
+  transformAttributesAssignmentsToJson,
+  transformStringAssignmentToJson,
+} from '../../shared/utils/uidl-utils'
+
+import { ComponentUIDL, UIDLNode } from '../../typings/uidl-definitions'
+
+export const parseComponentJSON = (input: Record<string, unknown>): ComponentUIDL => {
+  const node = input.node as Record<string, unknown>
```

Comment:
should we declare this Record<string, unknown> as RawInput or something like that? if we start using it in a lot of places it will be some repetition

Reply:
Seems a bit generic and not worth the import imo :-?